pandas/core/frame.py

```diff
@@ -1258,6 +1258,13 @@ def from_items(cls, items, columns=None, orient='columns'):
         """
         keys, values = lzip(*items)
 
+        import array
```

Comment:
@jreback : I feel like we must have some function checking if an object is array-like?

Reply:
isn't `is_list_like` enough in this case?

Reply:
``is_list_like`` seems to work fine. I'll update the PR.